package.json

```diff
@@ -14,85 +14,109 @@
+    "babel-runtime": "^6.20.0",
+    "camelcase": "^5.2.0",
+    "case-sensitive-paths-webpack-plugin": "2.2.0",
     "cf-buttons": "^4.3.0",
     "cf-core": "^4.2.0",
     "cf-expandables": "^4.1.2",
-    "cf-forms": "^10.2.0",
+    "cf-forms": "^10.2.1",
     "cf-grid": "^4.1.0",
-    "cf-icons": "^10.1.0",
+    "cf-icons": "^10.2.1",
     "cf-layout": "^4.2.0",
     "cf-pagination": "^4.1.1",
     "cf-tables": "^4.1.1",
     "cf-typography": "^4.1.0",
-    "history": "^4.6.1",
-    "moment": "^2.19.3",
-    "prop-types": "^15.5.10",
-    "query-string": "^5.0.0",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
-    "react-intl": "^2.4.0",
-    "react-modal": "^3.0.4",
-    "react-redux": "^5.0.5",
-    "react-router": "^4.2.0",
-    "react-router-dom": "^4.2.2",
-    "redux": "^3.7.0",
-    "redux-devtools-extension": "^2.13.2",
-    "redux-thunk": "^2.2.0"
-  },
-  "devDependencies": {
-    "autoprefixer": "7.1.5",
-    "babel-core": "6.26.0",
-    "babel-eslint": "8.0.1",
-    "babel-jest": "21.2.0",
-    "babel-loader": "7.1.2",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
-    "babel-preset-react-app": "^3.0.3",
-    "babel-runtime": "^6.20.0",
-    "case-sensitive-paths-webpack-plugin": "2.1.1",
     "chalk": "2.2.0",
     "connect-history-api-fallback": "1.4.0",
     "core-js": "^2.5.1",
-    "coveralls": "^3.0.1",
+    "coveralls": "^3.0.6",
     "cross-spawn": "5.1.0",
-    "css-loader": "0.28.7",
+    "css-loader": "2.1.1",
     "detect-port": "1.2.1",
-    "dotenv": "4.0.0",
+    "dotenv": "6.2.0",
+    "dotenv-expand": "4.2.0",
     "enzyme": "^3.1.0",
     "enzyme-adapter-react-16": "^1.0.1",
-    "eslint": "4.9.0",
-    "eslint-config-react-app": "^2.0.1",
-    "eslint-loader": "1.9.0",
-    "eslint-plugin-flowtype": "2.39.1",
-    "eslint-plugin-import": "2.8.0",
-    "eslint-plugin-jsx-a11y": "6.0.2",
-    "eslint-plugin-react": "7.4.0",
-    "extract-text-webpack-plugin": "3.0.1",
-    "file-loader": "1.1.5",
-    "fs-extra": "4.0.2",
-    "html-webpack-plugin": "2.30.1",
-    "http-proxy-middleware": "0.17.4",
+    "eslint": "^6.1.0",
+    "eslint-config-react-app": "^5.0.1",
+    "eslint-loader": "2.2.1",
+    "eslint-plugin-flowtype": "3.13.0",
+    "eslint-plugin-import": "2.18.2",
+    "eslint-plugin-jsx-a11y": "6.2.3",
+    "eslint-plugin-react": "7.14.3",
+    "eslint-plugin-react-hooks": "^1.6.1",
+    "file-loader": "3.0.1",
+    "fs-extra": "7.0.1",
+    "history": "^4.10.1",
+    "html-webpack-plugin": "4.0.0-beta.5",
+    "http-proxy-middleware": "^0.20.0",
+    "identity-obj-proxy": "3.0.0",
     "ignore-loader": "^0.1.2",
-    "jest": "^21.2.1",
+    "is-wsl": "^1.1.0",
+    "jest": "24.8.0",
     "jest-cli": "^21.2.1",
+    "jest-environment-jsdom-fourteen": "0.1.0",
+    "jest-resolve": "24.8.0",
+    "jest-watch-typeahead": "0.3.1",
     "json-loader": "0.5.7",
-    "less": "^3.0.0-alpha.3",
+    "less": "^3.10.3",
     "less-loader": "^4.0.5",
+    "lodash": "^4.17.15",
```

Comment:
@sephcoster adding a newer lodash doesnt kill the snyk warning

Reply:
I'm assuming that Snyk made their own patch for it because lodash hasn't patched it yet. If they had, Snyk would just recommend that you upgrade lodash.